Source/MDFastBindingBlueprint/Private/BlueprintExtension/MDFastBindingWidgetBlueprintExtension.cpp

```diff
@@ -77,6 +64,9 @@ void UMDFastBindingWidgetBlueprintExtension::HandleCopyTermDefaultsToDefaultObje
 		
 			// There's a chance we could perform some compile-time steps here that would improve runtime performance
 			CompilerContext->AddExtension(WidgetBPClass, BindingClass);
+
+			// The blueprint has been fully recompiled here, we need to update the binding graphs
```

Comment:
So does that mean that the diff tool compiles both versions of the blueprints being compared? That's surprising.